pkg/apis/resource/types.go

```diff
@@ -463,10 +469,97 @@ type DeviceRequest struct {
 	// +optional
 	// +featureGate=DRAAdminAccess
 	AdminAccess *bool
+
+	// FirstAvailableOf contains subrequests, exactly one of which must be satisfied
+	// in order to satisfy this request. This field may only be set in the
+	// entries of DeviceClaim.Requests.
+	//
+	// +optional
+	// +oneOf=deviceRequestType
+	// +listType=atomic
+	// +featureGate=DRAPrioritizedList
+	FirstAvailableOf []DeviceSubRequest
+}
+
+// DeviceSubRequest describes a request for device provided in the
+// claim.spec.devices.requests[].firstAvailableOf slice. Each
```

Comment:
Nit:

```suggestion
// claim.spec.devices.requests[].firstAvailableOf array. Each
```

"slice" is a Go thing. This documentation is more targeted towards users of the Kubernetes API, which is independent of Go.

Reply:
Fixed.